pycbc/conversions.py

```diff
@@ -1119,12 +1119,17 @@ def freqlmn_from_other_lmn(f0, tau, current_l, current_m, new_l, new_m):
         correspond to an unphysical Kerr black hole mass and/or spin,
         ``numpy.nan`` will be returned.
     """
-    mass = final_mass_from_f0_tau(f0, tau, l=current_l, m=current_m)
-    spin = final_spin_from_f0_tau(f0, tau, l=current_l, m=current_m)
+    mass = numpy.array(final_mass_from_f0_tau(f0, tau, l=current_l,
```

Comment:
Maybe you could use the function `ensurearray`, like it is done here https://github.com/gwastro/pycbc/blob/master/pycbc/conversions.py#L1045. So leaving these two lines as they were and then adding
`mass, spin, _ = ensurearray(mass, spin)`

Reply:
Yes, we could just do it in the same way as was used in that function - then the behaviour would be consistent, and it also already takes care of the output type.
I will add that.